docs/blog/0.11.md

```diff
@@ -0,0 +1,30 @@
+
+***Note*** that this version of Rill has a breaking change that requires you to recreate your duckDB database for existing projects.
+
+- **Error feedback during data ingestion —** Rill is powered by duckDB, a *fast* analytical database with [nice ergonomics around data ingestion](https://duckdb.org/docs/data/csv). Data imports make smart assumptions about the type of data contained in CSV files to create typed sources. Unfortunately, CSV data ingestion is notoriously error prone — from encoding errors to missing delimiters and mismatched columns. Enjoy more success getting data into Rill with this release as we surface better feedback in the application to help you troubleshoot what has gone wrong. Messages stay on the screen until you have a chance to read it, take action, or dismiss.
+- **Building on your last correct query —** There is no *“Run Button”* in Rill. Instead, we run the query you are writing in real time providing feedback with every keystroke. This means that you are often presented with an error as you type the next sequence of words to complete the SQL statement. Users have given us feedback that because you are in the flow of writing a query to change this output, keeping the last valid result set helps inform what to do next. We agree, and in this update you will always see the last valid result set along with the error between valid queries.
+- **Screen URL routes —** As you build out your Rill project you might create many sources, models, and dashboards that you want to revisit in the future. The local Rill Developer experience has all been served under one URL: [`http://localhost:8080/`](http://localhost:8080/). This means that if you want to see a specific asset, you need to click around to find it. In this update we introduce routes that allow you to navigate to a specific screen using only a URL. These changes lay the foundation for sharing specific Rill insights with others in the future.
+- **Upgrade to duckDB 0.5.0 —** We are excited to update your experience to include the new features released by duckDB in 0.5.0. Note that this is a breaking change where projects created with prior versions of Rill Developer will not be compatible.
+    - If you encounter this error when you try to start Rill: `Error: could not open database` you can fix your project by deleting `./stage.db`, `./stage.db.wal`, and `./stage/*`. The next time you run `rill start` in the CLI, a new updated database will be created. You will then need to reimport your data to restore existing models and dashboards.
```

Comment:
By `./stage/*` do you mean `./state/*`?

Reply:
I did! Good catch, I just changed it. Does not affect the release, just the docs and we just tweeted 20 min ago.